molgenis-api/src/main/java/org/molgenis/api/model/response/PageResponse.java

```diff
@@ -53,4 +53,8 @@ public PageResponse build() {
       return pageResponse;
     }
   }
+
+  private static int getTotalPages(int pageSize, int totalElements) {
```

Comment:
Did you consider making totalPages a derived property? (https://github.com/google/auto/blob/master/value/userguide/howto.md#memoize)